fix: MD033/no-inline-html

nschonni opened the pull request:
- Escape inline elements that didn't render
- Ignore elements currently used by Markdown tables
- Encode elements inside pre/code blocks

Lxiamail reviewed (commented):
Didn't see any technical concern with WCF related doc updates. Thanks

rpetrusha reviewed (changes requested):
Thanks, @nschonni. In the toc.md files, however, the backslash is rendering in the output TOC. And one change was unrelated to the original paragraph.

mairaw commented:
Unfortunately, the proper Markdown syntax never worked for the TOC files. 

Also, with PR #11443, you'd get a lot of merge conflicts.

nschonni commented:
I'll fix @rpetrusha's catch, and I'll just rebase to remove all the TOC changes once that other PR lands

nschonni committed:
fix: MD033/no-inline-html

- Escape inline elements that didn't render
- Ignore elements currently used by Markdown tables
- Encode elements inside pre/code blocks

nschonni commented on docs/framework/wcf/troubleshooting-the-getting-started-tutorial.md:
```diff
@@ -5,7 +5,7 @@ ms.assetid: 69a21511-0871-4c41-9a53-93110e84d7fd
 # Troubleshoot the Get started with Windows Communication Foundation tutorials
 
-This article provides solutions for the most common problems and errors you might face when you follow the steps in the [Tutorial: Get started with Windows Communication Foundation applications](getting-started-tutorial.md). 
+ Visual Studio saves project files in c:\users\\<user name\>\Documents\\<Visual Studio version\>\Projects.  
```
Not sure where that came from, but I undid the change

nschonni commented:
Actually, just removed the TOC changes since it was simple while reverting the other change

rpetrusha commented:
Thanks, @nschonni. I'll merge your PR now.
